corehq/apps/app_manager/models.py

```diff
@@ -1005,7 +1007,7 @@ def default_name(self):
         app = self.get_app()
         return trans(
             self.name,
-            [app.default_language] + app.build_langs,
+            [app.default_language] + app.langs,
```

Comment:
Do you know where `default_name` is used? If it's used during build, then you'll want it to use the languages in the profile rather than all languages.